mvpa2/tests/test_procrust.py

```diff
@@ -105,7 +105,10 @@ def test_simple(self, svd, oblique):
 
                 # Test if we get back
                 d_s_f_r = pm.reverse(d_s_f)
-
+                # Test if recon proj is true inverse except for high->low projection
+                if nf_s <= nf_t:
```

Comment:
No need. Since pinv(R) is same as R.T (by definition), they both should give the same matrix, albeit pinv() going through more computations.